ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -311,6 +311,11 @@ objects:
         # create A record for the api
         curl -v $CI_DNS_IP:8080/add -d '{"cluster_name": "'$CLUSTER_NAME'", "lb_fip": "'$LB_FIP'"}'
 
+        # for parallel jobs we need to use a bigger flavor, so here we're substiting it with another
```

Comment:
This in't really a safe sort of check.  Job names are not apis.  If you want to do this, define an environment variable parameter on your template and change your release jobs to use a larger flavor.

Reply:
In general I would expect your default flavor to be set so that you can successfully run the e2e suite.

Reply:
yeah, just noticed that... please check a new version